core/dbt/constants.py

```diff
@@ -1,9 +1,15 @@
+import os
+
 SECRET_ENV_PREFIX = "DBT_ENV_SECRET_"
 DEFAULT_ENV_PLACEHOLDER = "DBT_DEFAULT_PLACEHOLDER"
 METADATA_ENV_PREFIX = "DBT_ENV_CUSTOM_ENV_"
 
-MAXIMUM_SEED_SIZE = 1 * 1024 * 1024
-MAXIMUM_SEED_SIZE_NAME = "1MB"
+def get_max_seed_size():
+    mx = os.getenv('DBT_MAXIMUM_SEED_SIZE', '1')
+    return int(mx)
+
+MAXIMUM_SEED_SIZE = get_max_seed_size() * 1024 * 1024
+MAXIMUM_SEED_SIZE_NAME = str(get_max_seed_size()) + "MB"
```

Comment:
Let's make sure to use the [IEC standard abbreviation](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units) (i.e., MiB when using 1024 * 1024).
```suggestion
MAXIMUM_SEED_SIZE_NAME = str(get_max_seed_size()) + "MiB"
```

Reply:
I know you're just working with what was already there.

I'll probably open up a separate ticket to comb through this repo for places where we accidentally use MB instead of MiB.